src/style.css

```diff
@@ -1 +1,118 @@
-/* add styles here */
+/* #region scale */
+
+.page__body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+/* #endregion */
+
+/* #region gray stars */
+.stars__star {
+  display: block;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-left: 4px;
+}
+
+/* #endregion */
+
+/* #region stars--1 */
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-left: 4px;
+  display: block;
+  width: 16px;
+  height: 16px;
```

Comment:
```suggestion

```

Reply:
No need to repeat the same styling everywhere. It's enough to have this in the basic class